### PR TITLE
PMM-7 PMM v3 devcontainer builds

### PR DESCRIPTION
**Pull request by artemgavrilov**
PMM-7

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

**Review by JiriCtvrtka — APPROVED** [review]
I am just wondering for what purpose there are two tmp files?

**Comment by artemgavrilov** [comment]
> I am just wondering for what purpose there are two tmp files?

Already removed, added by accident)

File: docker-compose.yml [hunk 1]
```diff
@@ -131,7 +131,7 @@ services:
     depends_on:
       - ch
       - victoriametrics
-    image: ${PMM_CONTAINER:-perconalab/pmm-server:dev-container}
+    image: ${PMM_CONTAINER:-perconalab/pmm-server:3-3-3-dev-container}
```
Comment (ademidoff) [inline docker-compose.yml:134]: Should it be tripple?

Comment (artemgavrilov) [reply docker-compose.yml:134]: Nope. I wonder how it happened

Comment (artemgavrilov) [reply docker-compose.yml]: Fixed

**Review by ademidoff — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by surbhat1595 — APPROVED** [review]

**Pull request merged**